examples/sphere.py

```diff
@@ -210,177 +685,20 @@ def create_scheduler(algorithm,
-                seed=s,
-            ) for s in emitter_seeds
-        ]
-    elif algorithm == "cma_mega":
-        # Note that only one emitter is used for cma_mega. This is to be
-        # consistent with Fontaine 2021 <https://arxiv.org/abs/2106.03894>.
-        emitters = [
-            GradientArborescenceEmitter(
-                archive,
-                x0=initial_sol,
-                sigma0=10.0,
-                lr=1.0,
-                grad_opt="gradient_ascent",
-                selection_rule="mu",
-                bounds=None,
-                batch_size=batch_size - 1,  # 1 solution is returned by ask_dqd
-                seed=emitter_seeds[0])
-        ]
-    elif algorithm == "cma_mega_adam":
-        # Note that only one emitter is used for cma_mega_adam. This is to be
-        # consistent with Fontaine 2021 <https://arxiv.org/abs/2106.03894>.
-        emitters = [
-            GradientArborescenceEmitter(
-                archive,
-                x0=initial_sol,
-                sigma0=10.0,
-                lr=0.002,
-                grad_opt="adam",
-                selection_rule="mu",
-                bounds=None,
-                batch_size=batch_size - 1,  # 1 solution is returned by ask_dqd
-                seed=emitter_seeds[0])
-        ]
-    elif algorithm == "cma_mae":
-        emitters = [
-            EvolutionStrategyEmitter(
-                archive,
-                x0=initial_sol,
-                sigma0=0.5,
-                ranker="imp",
-                selection_rule="mu",
-                restart_rule="basic",
-                batch_size=batch_size,
-                seed=s,
-            ) for s in emitter_seeds
-        ]
-    elif algorithm in ["cma_maega"]:
-        emitters = [
-            GradientArborescenceEmitter(archive,
-                                        x0=initial_sol,
-                                        sigma0=10.0,
-                                        lr=1.0,
-                                        ranker="imp",
-                                        grad_opt="gradient_ascent",
-                                        restart_rule="basic",
-                                        bounds=None,
-                                        batch_size=batch_size,
-                                        seed=s) for s in emitter_seeds
-        ]
-
-    if algorithm == "me_map_elites":
-        scheduler = BanditScheduler(archive,
-                                    emitters,
-                                    num_active=12,
-                                    reselect="terminated")
-        scheduler_name = "BanditScheduler"
-    else:
-        scheduler = Scheduler(archive,
-                              emitters,
-                              result_archive=result_archive,
-                              add_mode=mode)
-        scheduler_name = "Scheduler"
+    num_emitters = sum(e["num_emitters"] for e in config["emitters"])
+    random.seed(10)
```

Comment:
Can you use numpy's rng here? e.g. `rng = np.default_rng()`, then call `rng.integers()` -- see here https://numpy.org/doc/stable/reference/random/generator.html